test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.elasticsearch.test.hamcrest;
 
+import org.apache.logging.log4j.util.Strings;
```

Comment:
You are right, I meant to use String.joing but ended up with the worng import. I will change that